docs/medical-api/api-reference/document/list-documents.mdx

```diff
@@ -7,11 +7,6 @@ api: "GET /medical/v1/document"
 This endpoint returns the document references available at Metriport which are associated with the
 given Patient.
 
-It also returns the status of querying document references across HIEs, indicating whether
-there is an asynchronous query in progress (status `processing`) or not (status `completed`).
-
-If the query is in progress, you will also receive the total number of documents to be queried as well as the ones that have already been completed.
-
```

Comment:
we should also remove
```
"download": {
    "status": "completed",
    "total": 100,
    "successful": 95,
    "errors": 5
  },
  "convert": {
    "status": "processing",
    "total": 20,
    "successful": 15,
    "errors": 1
  }
```

from the example response below

also, need to remove `<Snippet file="document-progress.mdx" />`

Reply:
good catch!